hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/processor/maxwell/MaxwellJsonKafkaSourcePostProcessor.java

```diff
@@ -49,12 +48,15 @@
  */
 public class MaxwellJsonKafkaSourcePostProcessor extends JsonKafkaSourcePostProcessor {
 
-  private static final Logger LOG = LogManager.getLogger(MaxwellJsonKafkaSourcePostProcessor.class);
-
   private static final ObjectMapper MAPPER = new ObjectMapper();
 
+  private final String databaseRegex;
+  private final String tableRegex;
+
   public MaxwellJsonKafkaSourcePostProcessor(TypedProperties props) {
     super(props);
+    databaseRegex = props.getString(Config.DATABASE_NAME_REGEX_PROP.key(), null);
+    tableRegex = props.getString(Config.TABLE_NAME_REGEX_PROP.key());
```

Comment:
as a common practice, we should prefer `Option<String>` over `null`

Reply:
> as a common practice, we should prefer `Option<String>` over `null`

done, Thanks for your time